src/core/comm/comm_nccl.cu

```diff
@@ -170,6 +171,27 @@ void register_tasks(Legion::Machine machine,
   }
 }
 
+bool needs_barrier()
+{
+  int32_t ver;
+  auto status = cuDriverGetVersion(&ver);
+  if (status != CUDA_SUCCESS) {
+    const char* error_string;
+    cuGetErrorString(status, &error_string),
```

Comment:
Should be semicolon here instead of comma?

Reply:
12aef5f92ecfcf6be0eefd0191b2e0a9e80bafc0